### ISSUE TITLE
margin loss

### ISSUE THREAD

**Issue by manuel-delverme** [manuel-delverme/option-baselines#4]
https://github.com/manuel-delverme/option-baselines/blob/60dfddc90c4d220461f5b89ce2b92e0391bb009c/option_baselines/aoc/aoc.py#L239

is it correct?

**Comment by manuel-delverme** [comment]
Disable termination loss

**Comment by manuel-delverme** [comment]
check the - sign on margin loss

**Comment by manuel-delverme** [comment]
print option logits/distribution probs
